### ISSUE TITLE
No output from unit test

### ISSUE THREAD

**Issue by ubrenaf** [OpenFAST/openfast#779]
Hello, I have compiled openfast via mingw on windows 10 and want to check if it works. But when running the unit tests in each module (openfast/build/unit_tests/module), for example beamdyn_utest, either by `start beamdyn_utest.exe` or just clicking, there is no output / results shown. According to [the documentation](https://openfast.readthedocs.io/en/dev/source/testing/unit_test.html#executing) I should get some feedback so I'm not sure what's wrong or if I missed something?

My build process was downloading using `git clone --single-branch --branch dev --recursive https:/...` and compiled with `cmake .. -G "MinGW Makefiles" -DBUILD_TESTING=ON` and then `mingw32-make`. I've tried recompiling the test separately by `make beamdyn_utest` but this had no effect. I don't see any errors when configuring, compiling or running either openfast.exe or the tests exe's.

**Comment by ubrenaf** [comment]
This resolved itself when using cmake option `-DBUILD_FASTFARM=ON` and `make FAST.Farm`. I was then able to run the tests. I assume the tests in dev only work for FAST.farm.

**Comment by andrew-platt** [comment]
I think you may have been executing the test incorrectly.  From a terminal window, the `beamdyn_utest` should be executed using `ctest beamdyn_utest` from the build directory.  Double clicking on it would have no effect.

The `-DBUILD_FASTFARM=ON` option should not affect the BeamDyn unit test in any way.  @rafmudaf, is it possible we have a logic error someplace?

**Comment by rafmudaf** [comment]
It's possible something was changes with FAST Farm, but I would expect this to show up in GitHub Actions. The `regress-tests-release` pipeline runs the BeamDyn unit tests with FAST Farm not enabled (https://github.com/OpenFAST/openfast/blob/dev/.github/workflows/automated-dev-tests.yml#L66).

**Comment by ubrenaf** [comment]
Yes I see now that you are right @andrew-platt, ctest can also run the unit tests. I think I just assumed otherwise since ctest was not mentioned in the unit test chapter (3.2.1) in the documentation. And yes, `-DBUILD_FASTFARM` has no effect.

But the unit tests can also be run individually directly through the executables. My very simple error was that either double clicking the .exe, or writing `start beamdyn_utest.exe`, in Windows both open a new command prompt which immediately closes upon completion. Instead you should write only `/path/to/beamdyn_utest.exe`. Or indeed `ctest beamdyn_utest` from build.

**Comment by rafmudaf** [comment]
I'm reopening this issue to keep alive to suggestion for improving unit test documentation. Thanks @ubrenaf for pointing this out.
